### PR TITLE
Majorly buffs standard holoparas

### PR DESCRIPTION
**Pull request by gumner**
list of the buffs:
chance of reflecting bullets
stuns you for 1 second while punching you (you stay still and dont drop yer weapon(i think you dont drop it))
has a low chance to stun you for 5 seconds
has a even lower chance to give you organ damage

### CODE REVIEW

**Comment by gumner** [comment]
holy fuck why is the delete jotaro commit still there

**Comment by Anderbergst** [comment]
ora ora ora ora ora

**Comment by gumner** [comment]
btw i made it so that every simple animal has the reflect projectile things,so if you want a simple mob to reflect shit,just give it a mob_reflect_chance , the default is 0

**Comment by asskek** [comment]
jojo station 13

**Comment by PopNotes** [comment]
According to all known laws of aviation, there is no way a bee should be able to fly. Its wings are too small to get its fat little body off the ground. The bee, of course, flies anyway because bees don't care what humans think is impossible.

**Comment by Kierany9** [comment]
>btw i made it so that every simple animal has the reflect projectile things,so if you want a simple mob to reflect shit,just give it a mob_reflect_chance , the default is 0

as soon as this is merged i am changing this var to be available for all mobs kthnx

**Comment by JohnGinnane** [comment]
Is this my reflection code or the old one that just makes projectiles go in random directions?

File: code/modules/mob/living/simple_animal/simple_animal.dm [hunk 1]
```diff
@@ -264,11 +269,30 @@
...
 
-/mob/living/simple_animal/bullet_act(obj/item/projectile/Proj)
-	if(!Proj)
+/mob/living/simple_animal/bullet_act(obj/item/projectile/P)
+	if(!P)
 		return
-	apply_damage(Proj.damage, Proj.damage_type)
-	Proj.on_hit(src)
+	if(prob(mob_reflect_chance))
+		visible_message("<span class='danger'>The [P.name] gets reflected by [src]!</span>")
```
Comment (ghost) [inline code/modules/mob/living/simple_animal/simple_animal.dm]: change [src] to [name]

**Comment by Pyko1** [comment]
Im not exactly happy with the /mob/living/simple_animal/hostile/guardian/punch/AttackingTarget() proc.

Should prolly start with.
```
..()
if(istype(target, /mob/living))
```

Also I understood you wanted the "standard punch" happen every time, and in top of that have a chance for extra. Now its standard punch damage or extra 1 or extra 2.

**Comment by blobbernaut** [comment]
This is sick

**Comment by Anderbergst** [comment]
incoming standard holo stunlock! i like it.

**Comment by ghost** [comment]
if this is all good, i like it the way it is. standard was waaay too weak.
+1

File: code/modules/mob/living/simple_animal/simple_animal.dm (hunk 1 above)
Comment (gumner) [reply code/modules/mob/living/simple_animal/simple_animal.dm]: yer sure? the message comes fine when i tested it

**Comment by gumner** [comment]
@JohnGinnane i ripped the reflection code of human_defense code

**Comment by gumner** [comment]
@McBawbaggings @spank gimme input pls

**Comment by gumner** [comment]
i have a fetish on labels

**Comment by gumner** [comment]
everyone commenting on other pr's,please comment on mine too,im quite depressed

**Comment by ArcLumin** [comment]
yeah okay comment comment comment

**Comment by SpankMasterson** [comment]
+1

**Comment by gumner** [comment]
i need code review so i can try to port this to rebase

**Comment by ArcLumin** [comment]
Code's good. Merging as head okays

**Pull request merged**